### ISSUE TITLE
Upload form: It is not possible to "deselect" a term in a multiple choice field

### ISSUE THREAD

**Issue by RKasprowicz** [digital-york/oasis#78]
If a user selects a term in a multiple choice field (e.g. Proficiency of learners), it is not possible for the user to then deselect the field - i.e. not select any field. For the option fields (under "Additional fields" the user must be able to decide not to select any field.

For example: I selected "Other" under "Proficiency of learners" and now I cannot deselect it.

![image](https://user-images.githubusercontent.com/36402489/37163520-ed33dcc4-22ef-11e8-87e0-664da7345c19.png)

**Comment by yf508** [comment]
@RKasprowicz  What we can do is:
* For mandatory fields: this bug doesn't apply as user needs to select at least an item before they can submit/save.
* For optional fields, we can have an empty field as shown below. Would it be ok?
![image](https://user-images.githubusercontent.com/437701/37335022-672e52e4-26a5-11e8-87a2-c87865704a70.png)

**Comment by RKasprowicz** [comment]
@yf508 I agree, leave an empty field at the top of the list, for optional fields.

**Comment by yf508** [comment]
This is related to #81, which has been fixed. So changing some of the fields from dropdown list to checkbox could avoid this problem.
